src/lib/cli.js

```diff
@@ -31,46 +31,67 @@ process.title = 'verdaccio';
 const logger = require('./logger');
 logger.setup(); // default setup
 
+const envinfo = require('envinfo');
 const commander = require('commander');
 const pkgVersion = module.exports.version;
 const pkgName = module.exports.name;
 
 commander
+  .option('--info', 'prints debugging information about the local environment')
```

Comment:
It is perfect by far, tested works fine, linting and test run well. But I forgot to mention this

As the other commands we provide the short and long version. I need this little change.`.option('-i, --info, ` ...`
so users can do `verdaccio -i` and `verdaccio --info`